float32 M # 진폭
float32 k # 기울기
float32 c # 시프트
float32 confidence # 신뢰도
